assets/js/blocks/rating-filter/utils.ts

```diff
@@ -17,3 +22,16 @@ export const getActiveFilters = ( queryParamKey = 'filter_rating' ) => {
 
 	return parsedParams;
 };
+
+export const parseAttributes = ( data: Record< string, unknown > ) => {
+	return {
+		heading: isString( data?.heading ) ? data.heading : '',
+		headingLevel:
```

Comment:
Since this is a new block, and users don't have a legacy `heading` attribute is this needed?